examples/api/test/material/bottom_app_bar/bottom_app_bar.2_test.dart

```diff
@@ -0,0 +1,72 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_api_samples/material/bottom_app_bar/bottom_app_bar.2.dart'
+    as example;
+import 'package:flutter_test/flutter_test.dart';
+
+void main() {
+  testWidgets('Floating Action Button visibility can be toggled',
+      (WidgetTester tester) async {
+    await tester.pumpWidget(const example.BottomAppBarDemo());
+
+    expect(find.byType(FloatingActionButton), findsOneWidget);
+
+    // Tap the switch to hide the FAB.
+    await tester.tap(find.byType(SwitchListTile).first);
+    await tester.pumpAndSettle();
+
+    expect(find.byType(FloatingActionButton), findsNothing);
+  });
+
+  testWidgets('BottomAppBar elevation can be toggled',
+      (WidgetTester tester) async {
+    // Build the app
```

Comment:
```suggestion
    // Build the app.
```
Please add a dot for other comments that miss one.